layout/_third-party/katex.swig

```diff
@@ -0,0 +1,7 @@
+{% if theme.katex.enable %}
+  {% if not theme.katex.per_page or (page.total or page.mathjax) %}
```

Comment:
This is just copy from `mathjax.swig` and add some modifications.

I use the `page.mathjax` to be compatible with those use mathjax previously.

It might be better to use another name such `math`

see the related PR https://github.com/iissnan/hexo-theme-next/pull/887